src/vivarium_ciff_sam/components/risk.py

```diff
@@ -269,6 +269,19 @@ def on_initialize_simulants(self, pop_data: SimulantData) -> None:
         self.population_view.update(exposure)
 
 
+class PreventativeZincSupplementation(Risk):
+
+    def __init__(self, risk: str):
+        super().__init__(risk)
+        self.propensity_column_name = f'therapeutic_zinc_propensity'
+
+    #################
+    # Setup methods #
+    #################
+
+    def _register_simulant_initializer(self, builder: Builder) -> None:
+        pass
+
```

Comment:
This is all you need to do to make this work, but you might notice that you are creating a `RandomnessStream` that is never used. You could override `_get_randomness_stream()` to return `None` to avoid that.